smartsim/_core/launcher/dragon/dragonBackend.py

```diff
@@ -55,25 +66,75 @@
     DragonUpdateStatusRequest,
     DragonUpdateStatusResponse,
 )
-from smartsim._core.utils.helpers import create_short_id_str
-from smartsim.status import TERMINAL_STATUSES, SmartSimStatus
+from ...._core.utils.helpers import create_short_id_str
+from ....log import get_logger
+from ....status import TERMINAL_STATUSES, SmartSimStatus
 
 DRG_ERROR_STATUS = "Error"
```

Comment:
Can we put these together?

```py
class DragonStatus(StrEnum):
  Error
  Running
```

would let us look at one place, avoid importing new independent statuses elsewhere, etc.

Reply:
Done